spec/beaker-hostgenerator/parser_spec.rb

```diff
@@ -114,15 +114,16 @@ module BeakerHostGenerator
 
       context 'When using arbitrary host settings' do
         it 'Supports arbitrary whitespace in values' do
-          expect( parse_node_info_token("64{k1=value 1,k2=v2,k3=  v3  }") ).
+          expect( parse_node_info_token("64{k1=value 1,k2=v2,k3=  v3  ,k4=[v4~ v5 ~v6]}") ).
```

Comment:
Using the `~` here since we are calling the `parse_node_info_token` directly and the comma replacement does not happen beforehand.